moment.parseFormat.js

```diff
@@ -4,213 +4,212 @@
   /* global define */
```

Comment:
You can remove lines 5-15 entirely, and start with something like

```js
module.exports = parseFormat

function parseFormat() {
  var dayNames =  [ 'Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
  // etc
}
```

Then at the end check if `global.moment` is defined, and if it is, add it like this

```ls
if (global.moment) {
  global.moment.parseFormat = parseFormat()
}
```

Does that make sense?